src/__tests__/config.test.ts

```diff
@@ -16,20 +16,24 @@ test('configure() overrides existing config values', () => {
     asyncUtilTimeout: 5000,
     defaultDebugOptions: { message: 'debug message' },
     defaultIncludeHiddenElements: true,
+    useLegacyQueries: false,
```

Comment:
I'm not really convinced by this option's name but I couldn't find a good one and since it's temporary I thought it's not that important but if you find something better I'll change it

Reply:
@pierrezimmermannbam pls check this idea how to tackle the breaking change in this and other PRs: #1241